picard/ui/options/plugins.py

```diff
@@ -33,10 +34,6 @@
 from picard.ui.ui_options_plugins import Ui_PluginsOptionsPage
 
 
-def cmp_plugins(a, b):
```

Comment:
I agree with @mineo. If we're making the same function twice, it would be better to make it once.

`fcmp = lambda x: locale.strxfrm(x[2])` is exactly the same as
```python
fcmp(x):
    return locale.strxfrm(x[2])
```
And can be used in the same manner, but the latter has the benefit of being unittestable and docstringable.